components/infobox/commons/infobox_person.lua

```diff
@@ -24,187 +23,211 @@ local function GetBirthAndDeath()
-	infobox:cell('Nicknames', args.nicknames)
-	infobox:cell('Total Earnings', earnings)
-	self:addCustomCells(infobox, args)
 
-	local links = Links.transform(args)
-	local achievements = self:getAchievements(infobox, args)
-	local history = self:getHistory(infobox, args)
-
-	infobox:header('Links', not Table.isEmpty(links))
-	infobox:links(links, _LINK_VARIANT)
-	infobox:header('Achievements', achievements)
-	infobox:centeredCell(achievements)
-	infobox:header('History', history)
-	infobox:centeredCell(history)
-	infobox:centeredCell(args.footnotes)
-	self:addCustomContent(infobox, args)
-	infobox:bottom(self:createBottomContent(infobox, args))
-
-	if _SHOULD_STORE_DATA then
-		args.birthDisplay = birthDisplay
-		self:getCategories(infobox, args, role.category, status.store)
-
-		links = Links.makeFullLinksForTableItems(links, _LINK_VARIANT)
-		local lpdbData = {
-			id = args.id or mw.title.getCurrentTitle().prefixedText,
-			alternateid = args.ids,
-			name = args.romanized_name or args.name,
-			romanizedname = args.romanized_name or args.name,
-			localizedname = args.name,
-			nationality = args.country or args.nationality,
-			nationality2 = args.country2 or args.nationality2,
-			nationality3 = args.country3 or args.nationality3,
-			birthdate = birthday,
-			deathdate = deathday,
-			image = args.image,
-			region = args.region,
-			team = args.teamlink or args.team,
-			status = status.store,
-			type = role.store,
-			earnings = earnings,
-			links = links,
-			extradata = {},
-		}
-		lpdbData = self:adjustLPDB(lpdbData, args, role, status)
-		lpdbData.extradata = mw.ext.LiquipediaDB.lpdb_create_json(lpdbData.extradata)
-		lpdbData.links = mw.ext.LiquipediaDB.lpdb_create_json(lpdbData.links)
-		local storageType = self:getStorageType(args, role, status)
-
-		mw.ext.LiquipediaDB.lpdb_player(storageType .. self.name, lpdbData)
+	local widgets = {
+		Header{name = self:nameDisplay(args), image = args.image},
+		Center{content = {args.caption}},
+		Title{name = (args.informationType or 'Player') .. ' Information'},
+		Cell{name = 'Name', content = {args.name}},
+		Cell{name = 'Romanized Name', content = {args.romanized_name}},
+		Cell{name = 'Birth', content = {birthDisplay}},
+		Cell{name = 'Died', content = {deathDisplay}},
+		Customizable{id = 'status', children = {
+			Cell{name = 'Status', content = { args.status }
+				}
+			}
+		},
+		Customizable{id = 'roleorfaction', children = {
+			Cell{name = 'Role', content = { args.role }
+				}
+			}
+		},
+		Cell{
+			name = 'Country',
+			content = self:_createLocations(args, personType.category)
+		},
+		Cell{name = 'Region', content = {
+				self:_createRegion(args.region)
+			}
+		},
+		Cell{name = 'Team', content = {
+				self:_createTeam(args.team, args.teamlink),
+				self:_createTeam(args.team2, args.teamlink2)
+			}
+		},
+		Cell{name = 'Clan', content = {
+				self:_createTeam(args.clan, args.clanlink),
+				self:_createTeam(args.clan2, args.clanlink2)
+			}
+		},
```

Comment:
Can be removed I think? Seems very wiki specific

Reply:
i found it on all the wikis i checked to get a feel for which fields should be on commons
* RL
* CS
* SC
* SC2
* Warcraft
* dota2
* lol

Reply:
It is indeed in many of these, but almost none of them actually use it

Reply:
i am open to kicking it, but we probably should discuss it on discord^^

Reply:
kicked clans and made teams customizable (so clans can be re-added for e.g. broodwar who use it on >500 pages)